components/cam/src/dynamics/se/inidat.F90

```diff
@@ -109,6 +118,40 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
       end if
     end if
 
+!   Determine column closest to SCM point
+    if (single_column) then
+      if (scmlon .lt. 0._r8) then
+        scmposlon=scmlon+360._r8
+      else
+        scmposlon=scmlon
+      endif 
+      minpoint=10000.0_r8
+      ie_scm=1
+      i_scm=1
+      j_scm=1
+      indx_scm=1
+      do ie=1, nelemd
+        indx=1
+        do j=1, np
+          do i=1, np
+            testlat=elem(ie)%spherep(i,j)%lat * rad2deg
+            testlon=elem(ie)%spherep(i,j)%lon * rad2deg
+            if (testlon .lt. 0._r8) testlon=testlon+360._r8
+            testval=abs(scmlat-testlat)+abs(scmposlon-testlon)
```

Comment:
Would it be more accurate to use the root square difference to get the distance? `testval=( (scmlat-testlat)**2+(scmposlon-testlon)**2 ) ** 0.5`?